vuln/npm/498.json

```diff
@@ -0,0 +1,24 @@
+{
+  "id": 498,
+  "title": "Information Exposure Through Directory Listing",
+  "overview": "[harp] File access even when they have been set to be ignored.",
+  "created_at": "2018-12-02",
+  "updated_at": "2019-04-06",
+  "publish_date": "2019-04-06",
+  "author": {
+    "name": "Skynet",
+    "website": null,
+    "username": "skyn3t"
+  },
+  "module_name": "harp",
+  "cves": [],
+  "vulnerable_versions": "<=0.29.0",
```

Comment:
Module is currently at `0.30.0`. Does `0.30.0` resolve the issue? If so we may want to update the recommendation.

Reply:
Correct, it's not actually a mistake. See this: https://github.com/nodejs/security-wg/issues/526

Reply:
I will resolve so we can commit this in and work the details out in #526. Overall this issue is affecting other reports as well anyway.